lib/aasm/configuration.rb

```diff
@@ -28,5 +28,11 @@ class Configuration
 
     # namespace reader methods and constants
     attr_accessor :namespace
+
+    # These are the methods that will be override in a sub-class.
+    attr_accessor :override_methods
+
+    # Ignore all warnings, to be used at your own risk.
+    attr_accessor :ignore_override_warnings
```

Comment:
Perhaps we even get rid of it and not even allow a slippery slope here?

Let me get a touch commit and see what people think